core/src/main/scala/kafka/controller/TopicDeletionManager.scala

```diff
@@ -166,12 +166,12 @@ class TopicDeletionManager(config: KafkaConfig,
    * 2. partition reassignment in progress for some partitions of the topic
    * @param topics Topics that should be marked ineligible for deletion. No op if the topic is was not previously queued up for deletion
    */
-  def markTopicIneligibleForDeletion(topics: Set[String]): Unit = {
+  def markTopicIneligibleForDeletion(topics: Set[String], reason: => String): Unit = {
```

Comment:
Seemingly we don't need the by name parameter here since we only pass a literal string. Did you do it this way to make it more robust if the callers do some string concat?

Reply:
Oh yeah, I guess I did it sort of automatically. Is there a downside to having it?